src/scripts/main.js

```diff
@@ -354,7 +354,40 @@
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+'use strict';
+
+const table = document.querySelector('.dashboard');
+
+function createCell(content) {
+  const cell = document.createElement('td');
+  cell.textContent = content;
+  return cell;
+}
+
+function calculateAge(person) {
+  return person.died - person.born;
+}
+
+function calculateCentury(year) {
+  return Math.ceil(year / 100 - 1);
+}
```

Comment:
The `calculateCentury` function is clear and concise. However, the function name could be more descriptive. Consider renaming it to something like `getCenturyFromYear` to make it clear what the function does.